src/templates/partials/redis.hbs

```diff
@@ -0,0 +1,31 @@
+# {{output.header}}
+# {{{output.link}}}
+
+# Running Redis with TLS requires you to build Redis specifically for TLS
+# This can be done when making redis using "make && make BUILD_TLS=yes"
+
+################################# TLS/SSL #####################################
+port 0
+tls-port 6379
+tls-cert-file redis.crt
+tls-key-file redis.key
+tls-ca-cert-file ca.crt
+tls-ca-cert-dir /etc/ssl/certs
+tls-auth-clients yes
+tls-replication yes
+tls-cluster yes
+{{#unless (includes "old" form.config)}}
```

Comment:
The site should automatically warn users if this is the case, so this shouldn't be necessary.